cranelift/entity/src/list.rs

```diff
@@ -482,6 +482,46 @@ impl<T: EntityRef + ReservedValue> EntityList<T> {
         }
     }
 
+    /// Copies a slice from an entity list in the same pool to a position in this one.
```

Comment:
Could you add a note to the doc comment that the function will panic if `other` is the same as `self`?

Reply:
Done!